mobile/lib/components/shadowed_container.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+
+class ShadowedContainer extends StatelessWidget {
```

Comment:
here add doc comments, so the container can be easily reusable by other devs

Reply:
good suggestion